docker/build-farmOS.sh

```diff
@@ -61,16 +53,3 @@ allowedPlugins=(
 for plugin in ${allowedPlugins[@]}; do
   composer config --no-plugins allow-plugins.$plugin true
 done
-
-# Run composer install with optional arguments passed into this script.
-if [ $# -eq 0 ]; then
-  composer install
-else
-  composer install "$*"
-fi
```

Comment:
Remind me why we support passing optional arguments here?

Reply:
It is only used for passing `--no-dev` in production builds: https://github.com/farmOS/farmOS/blob/1bcc2942168e390797ee474c5bb914d22c66fc63/docker/Dockerfile#L63